docs/encyclopedia/temporal-nexus-overview.mdx

```diff
@@ -0,0 +1,118 @@
+
+**Nexus API Registry:** A `Nexus API Registry` is a list of Nexus `Endpoints` available within a Temporal environment. 
+Adding a Nexus `Endpoint` allows Nexus Operation requests to be sent to the Worker listening on the Endpoint’s target Namespace and Task Queue, from a caller Workflow using the name of the Endpoint in the Nexus API Registry.
+
+**Nexus Endpoints:** A Nexus `Endpoint` is like an API proxy that can serve one or more Nexus `Services` from a `Worker` listening on a target Namespace and Task Queue. 
+This enables presenting a clean API contract and hiding the underlying Temporal implementation, which may consist of internal `Workflows`, `Signals`, and other Temporal primitives – or arbitrary code. 
+In Temporal Cloud an endpoint’s access-control policy determines which callers are allowed to access a Nexus `Endpoint`.
+
+**Nexus Services:** A Nexus `Service` is a named collection of arbitrary-duration Nexus `Operations` that provide an API contract suitable for sharing across team boundaries. 
+Nexus Services are registered with a `Worker` that is listening on the target Namespace and Task Queue for an `Endpoint`.
+
+**Nexus Operations:** A Nexus `Operation` is not intrinsically durable, but can be made durable by backing it with Temporal primitives like a `Workflow`, `Update`, or `Signal`.
+The Temporal SDK provides helper functions to create Nexus Operations that may be registered with a Nexus Service in a `Worker`. 
+Synchronous Nexus `Operations` can also execute arbitrary code, but unlike `Activities` they have a short request timeout (under 10 seconds) and don’t support [heartbeats](https://community.temporal.io/t/best-practices-for-long-running-activities/934).
+
+Nexus RPC is designed for arbitrary-duration operations whose lifetime may extend beyond a traditional RPC and is an underpinning to connect durable executions within and across Namespaces, clusters and regions – with a clean API contract to streamline multi-team collaboration. Any service can be exposed via a Nexus endpoint as a set of sync or async Nexus operations – the latter provides an operation identity and a uniform interface to get the status of an operation or its result, receive a completion callback, or cancel the operation.
+
+Nexus Services and Operations are often registered in the same Worker as the underlying Temporal primitives they abstract, and use the same task-queue worker polling architecture as Workflows and Activities.
+
+To make a Nexus Service available for others to use, a Nexus Endpoint must be created in the Nexus API Registry with a target Namespace and task-queue. 
+In pre-release, all Nexus requests will be sent to the single target Namespace and task-queue for the Endpoint. Future releases may support multiple Endpoint targets, similar to how an API proxy can have multiple targets.
```

Comment:
I would just drop this for now, I don't think it provides value here. (And by the time we publish this, we'll be in public preview).